src/libstd/ffi/os_str.rs

```diff
@@ -312,12 +312,19 @@ impl Default for OsString {
 }
 
 #[stable(feature = "rust1", since = "1.0.0")]
-impl Debug for OsString {
-    fn fmt(&self, formatter: &mut fmt::Formatter) -> Result<(), fmt::Error> {
+impl fmt::Debug for OsString {
+    fn fmt(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
         fmt::Debug::fmt(&**self, formatter)
     }
 }
 
+#[stable(feature = "rust1", since = "1.20.0")]
+impl fmt::Display for OsString {
```

Comment:
This did not exist before and is a new stable feature being added in this PR, is that intentional? It was the libs team intention that this impl did *not* exist.